archiver.go

```diff
@@ -1440,17 +1440,17 @@ func DeleteArchivedRuns(ctx context.Context, config *Config, db *sqlx.DB, s3Clie
 	defer rows.Close()
 
 	var runID int64
-	var isActive bool
+	var status string
 	runCount := 0
 	runIDs := make([]int64, 0, archive.RecordCount)
 	for rows.Next() {
-		err = rows.Scan(&runID, &isActive)
+		err = rows.Scan(&runID, &status)
 		if err != nil {
 			return err
 		}
 
 		// if this run is still active, something has gone wrong, throw an error
-		if isActive {
+		if status != "C" && status != "E" && status != "I" && status != "F" {
```

Comment:
I think simpler to say `status == "A" || status == "W"` (i.e. run is active or waiting)

Reply:
I thought new status should be explicitly added here so that we do not mistakenly archive a future status we should not have archived

Reply:
It seems more likely we add a new status which should be included, since any new status is more likely to be some sort of exit type

Reply:
Who knows? Let me change that anyway